thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/analysis-options.ftl

```diff
@@ -0,0 +1,47 @@
+<div class="analysis-card padding-all">
+    <div class="analysis-title bottom-buffer">Trend Analysis
+      <span class="analysis-details">
+        (for <label class="label-medium-semibold">Metric</label> {{metricName}} from <label class="label-medium-semibold">Dataset</label> {{dataset}})
+      </span>
+    </div>
+    <div class="analysis-options">
+      <div class="analysis-options__datepicker">
+      <div class="datepicker-field">
+        <label class="label-medium-semibold">Date Range (Current) </label>
+        <div id="current-range" class="datepicker-range">
+          <span></span><b class="caret"></b>
```

Comment:
Wondering why add empty `<span>`? Also, perhaps consider using HTML entities for caret instead https://dev.w3.org/html5/html-author/charref?